### DISCUSSION TITLE
Is this assert meant to be before the allocation here?

### DISCUSSION THREAD

**Discussion by thendrix** [jrouwe/JoltPhysics#824] [General]
LockFreeHashMap.inl:119

https://github.com/jrouwe/JoltPhysics/blob/e9b4789c5d51cfb25326386c2e410b584eb87fae/Jolt/Core/LockFreeHashMap.inl#L122C34-L122C34

**Comment by jrouwe** [comment]
Yes it is, the LockFreeHashMap is only supposed to be initialized with ```Init()``` once. If this assert triggers, there will be a memory leak.
